### PR TITLE
Update renovate-config.js

### PR DESCRIPTION
**Pull request by Tugark**

### CODE REVIEW

File: renovate-config.js [hunk 1]
```diff
@@ -4,10 +4,7 @@ module.exports = {
     repositories: [
         'gisktzh/gb3-web_ui',
         'gisktzh/gb3-search_elastic',
-        // TODO: The following three projects are not yet publicly available
-        //       Therefore, 'auto-merge' is not possible making Renovate quite noisy as all PRs must be merged manually
-        // 'gisktzh/gb3-monitoring',
-        // 'gisktzh/gb3-cms',
+        'gisktzh/gb3-cms',
         // 'gisktzh/gb3-testcases',
```
Comment (wendlans) [inline renovate-config.js]: Add this project as well. Renovate will automatically create an onboarding PR.

File: renovate-config.js [hunk 2]
```diff
@@ -4,10 +4,7 @@ module.exports = {
...
         'gisktzh/gb3-search_elastic',
-        // TODO: The following three projects are not yet publicly available
-        //       Therefore, 'auto-merge' is not possible making Renovate quite noisy as all PRs must be merged manually
-        // 'gisktzh/gb3-monitoring',
-        // 'gisktzh/gb3-cms',
-        // 'gisktzh/gb3-testcases',
+        'gisktzh/gb3-cms',
+        'gisktzh/gb3-testcases',
     ],
-};
```
Comment (wendlans) [inline renovate-config.js:13]: Just wondering: why was the last empty line removed? It seems there is some different behavior between our IDEs

**Review by wendlans — APPROVED** [review]
All in all: this change looks perfect :D

File: renovate-config.js (hunk 2 above)
Comment (Tugark) [reply renovate-config.js:13]: I did it in the gui :)

**Pull request merged**